README.md

```diff
@@ -1,62 +1,46 @@
-`./setup.sh github.com/awesome-org/conduit-connector-file`)
-* (Optional) Set the code owners (in the `CODEOWNERS` file)
-
-With that, you're all set up and ready to start working on your connector! As a next step, we recommend that you 
-check out the [Conduit Connector SDK](https://github.com/ConduitIO/conduit-connector-sdk), which is the Go software 
-development kit for implementing a connector for Conduit.
-
-
-### Repository settings
-Following is a list of repository settings we recommend having.
-
-#### General/Pull Requests
-1. Allow squash merging only.
-2. Always suggest updating pull request branches.
-3. Allow auto-merge.
-4. Automatically delete head branches.
-
-#### Branch protection rules
-Protect the default branch using the following rules:
-1. Require a pull request before merging. Require approvals.
-2. Require status checks to pass before merging. Require branches to be up to date before merging.
-3. Specify status checks that are required.
-4. Require conversation resolution before merging.
-5. Do not allow bypassing the above settings.
-
-#### Actions/General
-1. Allow all actions and reusable workflows.
-2. Allow GitHub Actions to create and approve pull requests (if dependabot is used and it's configured to automatically
-merge pull requests).
-
-### Specification
-The `spec.go` file provides a programmatic representation of the configuration options.
-
-### Configuration
-This template provides two types of "configs":
-* general configuration (that applies to both, sources and destinations)
-* and source/destination specific configs.
-
-All are defined in `config.go` and are represented by separate types. General configs should be added to `pinecone.Config`,
-whereas any source or destination specific configs should be added to `pinecone.SourceConfig` and 
-`pinecone.DestinationConfig` respectively.
+# Conduit Connector for Pinecone
+
+The Pinecone connector is one of [Conduit](https://github.com/ConduitIO/conduit) builtin plugins. It provides both, a source and a destination connector for [Pinecone](https://www.Pinecone.io/).
+
+It uses the [grpc go Pinecone client](github.com/Pinecone-io/go-Pinecone) to connect to Pinecone.
+
+## What data does the OpenCDC record consist of?
+
+| Field                   | Description                                                                             |
+|-------------------------|-----------------------------------------------------------------------------------------|
+| `record.Operation`      | which conduit operation does the record do.                                             |
+| `record.Metadata`       | a string to string map representing the Pinecone vector metadata.                       |
+| `record.Key`            | the vector id that is being updated.                                                    |
+| `record.Payload.Before` | <empty>                                                                                 |
```

Comment:
Will "Before" always be empty under all circumstances? (thinking of updates for example)

Reply:
@raulb maybe I should've written discarded? I'm not using the `Before` field.